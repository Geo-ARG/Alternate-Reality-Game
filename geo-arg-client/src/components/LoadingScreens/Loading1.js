import React, { Component } from 'react';
import {
  StyleSheet,
  View,
  Image,
  Text,
} from 'react-native';

export default class Loading1 extends Component {
  componentDidMount(){
    setTimeout(()=>{
      this.props.navigator.push({page:"login"})
    }, 3000);
  }

  render() {
    return (
      <View style={styles.container}>
        <Image source={require("../../assets/loading.gif")} style={styles.imgHome} />
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
  },
  imgHome: {
    flex: 1,
    width: null,
    height: null,
  },
  buttonHome: {
    height: 60,
  },
  todos: {
    fontSize: 20,
    textAlign: 'center',
    marginTop: 10,
  },
  searchForm: {
    marginTop: 20,
  }
});
