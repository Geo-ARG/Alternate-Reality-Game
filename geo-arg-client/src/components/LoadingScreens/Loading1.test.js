import renderer from 'react-test-renderer';
import React from 'react'
import Loading1 from './Loading1'

test('test', () => {
  const tree = renderer.create(
      <Loading1 />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
