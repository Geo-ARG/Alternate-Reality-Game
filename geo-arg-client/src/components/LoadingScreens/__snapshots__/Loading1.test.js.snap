// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Image
    source={1}
    style={
      Object {
        "flex": 1,
        "height": null,
        "width": null,
      }
    }
  />
</View>
`;
